examples/vertex_mlops_enterprise/src/tfx_pipelines/training_pipeline.py

```diff
@@ -0,0 +1,320 @@
+
+    # Add dependency from example_validator to transform.
+    transform.add_upstream_node(example_validator)
+
+    # Get the latest model to warmstart
+    warmstart_model_resolver = Resolver(
+        strategy_class=latest_artifacts_resolver.LatestArtifactsResolver,
+        latest_model=Channel(type=standard_artifacts.Model),
+    ).with_id("WarmstartModelResolver")
+
+    # Model training.
+    trainer = Trainer(
+        module_file=TRAIN_MODULE_FILE,
+        examples=transform.outputs["transformed_examples"],
+        schema=schema_importer.outputs["result"],
+        base_model=warmstart_model_resolver.outputs["latest_model"],
+        transform_graph=transform.outputs["transform_graph"],
+        hyperparameters=hyperparams_gen.outputs["hyperparameters"],
+    ).with_id("ModelTrainer")
+    
+    if config.TRAINING_RUNNER == "vertex":
+        trainer = VertexTrainer(
+            module_file=TRAIN_MODULE_FILE,
+            examples=transform.outputs["transformed_examples"],
+            schema=schema_importer.outputs["result"],
+            base_model=warmstart_model_resolver.outputs["latest_model"],
+            transform_graph=transform.outputs["transform_graph"],
+            hyperparameters=hyperparams_gen.outputs["hyperparameters"],
+            custom_config=config.VERTEX_TRAINING_CONFIG
+        ).with_id("ModelTrainer")
+        
+
+    # Get the latest blessed model (baseline) for model validation.
+    baseline_model_resolver = Resolver(
+        strategy_class=latest_blessed_model_resolver.LatestBlessedModelResolver,
+        model=Channel(type=standard_artifacts.Model),
+        model_blessing=Channel(type=standard_artifacts.ModelBlessing),
+    ).with_id("BaselineModelResolver")
+
+    # Prepare evaluation config.
+    eval_config = tfma.EvalConfig(
+        model_specs=[
+            tfma.ModelSpec(
+                signature_name="serving_tf_example",
+                label_key=features.TARGET_FEATURE_NAME,
+                prediction_key="probabilities",
+            )
+        ],
+        slicing_specs=[
+            tfma.SlicingSpec(),
+        ],
+        metrics_specs=[
+            tfma.MetricsSpec(
+                metrics=[
+                    tfma.MetricConfig(class_name="ExampleCount"),
+                    tfma.MetricConfig(
+                        class_name="BinaryAccuracy",
+                        threshold=tfma.MetricThreshold(
+                            value_threshold=tfma.GenericValueThreshold(
+                                lower_bound={"value": float(config.ACCURACY_THRESHOLD)}
+                            ),
+                            # Change threshold will be ignored if there is no
+                            # baseline model resolved from MLMD (first run).
+                            change_threshold=tfma.GenericChangeThreshold(
+                                direction=tfma.MetricDirection.HIGHER_IS_BETTER,
+                                absolute={"value": -1e-10},
```

Comment:
Negative accuracy threshold? I think this should be some minimum value for which the model is acceptable (e.g. 0.7, 0.8, etc)